test/WebSites/RoutingWebSite/StartupWith21Compat.cs

```diff
@@ -14,11 +19,16 @@ public class StartupWith21Compat
         // Set up application services
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddMvc()
+            services
+                .AddMvc()
                 .SetCompatibilityVersion(CompatibilityVersion.Version_2_1);
 
             services.AddScoped<TestResponseGenerator>();
             services.AddSingleton<IActionContextAccessor, ActionContextAccessor>();
+
+            var actionDescriptorProvider = new RemoveControllerActionDescriptorProvider(typeof(EndpointRoutingController));
```

Comment:
Old routing doesn't support parameters in attributes so remove its actions to avoid exception